Update documentation related to running on mainnet (#795)
input-output-hk/hydra: docs/docs/getting-started/troubleshooting.md

### hydra-node

- Hydra node crashes after a fork

    + It ocurs during a fork and expects the operator to restart its hydra-node.

    + Reference: [issue #560](https://github.com/input-output-hk/hydra/issues/560) Hydra node crashed after a fork.
    
    + Workaround: Restarting your node should be enough to come back to live.

> 

Reviewer: I would not remove this item as it looks like this removal is unrelated to this branch purpose

Author: It is related since this info is outdated now. We merged yesterday what we hope is a fix for this so unless we see this happening again we consider it fixed.

>   - Only one or no utxo can be committed by each party to a Head.

Reviewer: ```suggestion
  - Each party can only commit zero or one utxo into a Head.
```

- Comitting a `UTxO` with a `ReferenceScript` will make the head not finalizable.
  - The protocol can only handle a maximum number of participants by Head. See [cost of collectcom transaction](https://hydra.family/head-protocol/benchmarks/transaction-cost/#cost-of-collectcom-transaction) or the `hydra-node` will inform you of the current configured maximum when trying to configure too many peers.

   + The `hydra-node` does prevent this from happening, but transactions in the
     Head protocol done by other implementations may result in the Head be not
     finalizable by `hydra-node`.
  - Each party can only commit zero or one utxo into a Head.

   + Creating output with reference scripts on the layer 2 ledger is
     non-problematic.
  - The head cannot be finalized if holding more than ~100 assets. See [cost of fanout transaction](https://hydra.family/head-protocol/benchmarks/transaction-cost/#cost-of-fanout-transaction) for latest numbers.

   + If you run into this, file an issue and the Hydra team will be able to help

>      you finalize your head in a manual workaround.

Reviewer 2: @v0d1ch Why was this point about reference scripts removed? This is still relevant!
